VMWare/vmware-esxi/tests/AUTH_PUBLIC_KEY.json

```diff
@@ -0,0 +1,37 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": "Accepted publickey for root from 1.2.3.4 port 33398 ssh2: RSA SHA256:qzkqJPyBUdiJUC4i/wbJzkYLTcUWwXRArUT90bUw2E0"
```

Comment:
Maybe we can extract the publickey in a custom field with the hash of the key and the algorithm of the hash.